mne_qt_browser/_pg_figure.py

```diff
@@ -1574,14 +1574,6 @@ def closeEvent(self, event):
                 self.mne.child_figs.remove(self)
         event.accept()
 
-    # If this widget gets activated (e.g., the user clicks away from the
-    # browser but then returns to it by clicking in a selection window),
-    # the main window should be raised as well
-    def event(self, event):
-        if event.type() == QEvent.WindowActivate:
-            _qt_raise_window(self.weakmain())
-        return super().event(event)
-
```

Comment:
@larsoner I can't reproduce the problem you describe (at least not on windows). Does it still

https://github.com/mne-tools/mne-qt-browser/assets/46245704/70cf02d7-b44d-4f1a-b97d-72dd84608025

 happen for you?

The problem is, that this apparently prevents Checkboxes in the Dialog from working, because when they are clicked the focus is forced back to main and they can't trigger and change state.

Reply:
It used to happen to me on macOS but doesn't anymore. So I'd say let's remove it and if it turns out to be a problem we'll find a better fix